docs/social-authentication.rst

```diff
@@ -2,6 +2,34 @@ Setting up Social Authentication via OAuth2 Providers
 =====================================================
 
 
+Auth0 OAuth2 Configuration
+--------------------------
+
+In the same way as with other Identiy-Providers, it's now possible to leverage Auth0 to authenticate users on DefectDojo.
+
+1. Inside your Auth0 dashboard create a new application (Applications / Create Application / Single Page Web Application).
+
+2. On the new application set the following fields:
+
+   * Name: "Defectdojo"
+   * Allowed Callback URLs: "http://the_hostname_you_have_dojo_deployed:your_server_port/complete/auth0/"
```

Comment:
nit-picking: https instead?

Reply:
copy paste issue, I adapted also the other references to https instead of http.